seed/agents.js

```diff
@@ -2,8 +2,8 @@ db.dropDatabase();
 
 var agents = [
   { extension: 'Brodo',   phoneNumber: '+15552483591' },
-  { extension: 'Dugobah', phoneNumber: '+15558675309' },
-  { extension: '113',     phoneNumber: '+15553185602' }
+  { extension: 'Dagobah', phoneNumber: '+15558675309' },
+  { extension: 'Oober',     phoneNumber: '+15553185602' }
```

Comment:
To align or not to align, that's the question. _Keep the alignment consistent please_.